meckenheim

```diff
@@ -7,20 +7,23 @@ networks:
   ipv4:
     - 10.152.64.0/18
   ipv6:
-    - fda4:1721:8451::/48
-#bgp:
-#  meckenheim1:
-#    ipv4: 10.207.0.152
-#    ipv6: fec0::a:cf:0:98
-#  meckenheim2:
-#    ipv4: 10.207.1.152
-#    ipv6: fec0::a:cf:1:98  
+    - 2a03:2260:301a::/48
 domains:
-  - ffmck
+  - ffvor
   - 64.152.10.in-addr.arpa
-  - 1.5.4.8.1.2.7.1.4.a.d.f.ip6.arpa
+  - a.1.0.3.0.6.2.2.3.0.a.2.ip6.arpa
 nameservers:
-  - 10.152.64.10
-  - 10.152.80.10
-  - fda4:1721:8451:100::1
-  - fda4:1721:8451:200::1
+  - 10.152.64.2
+  - 10.152.64.3
+  - 10.152.80.2
+  - 10.152.80.3
+  - 2a03:2260:301a::1
+  - 2a03:2260:301a::2
```

Comment:
Those nameservers will be unreachable unless you have bgp peers.

Reply:
But communities that use mkdns will use those addresses in their resolvers and won't reach your nameservers then.

Reply:
Ping